### PR TITLE
Feature/shooter.v4

### PR DESCRIPTION
**Pull request by KenyonSchep**
replaces #47

### CODE REVIEW

File: src/main/include/RobotContainer.h [hunk 1]
```diff
@@ -20,6 +20,7 @@
 #include "subsystems/Winch.h"
 #include "subsystems/DooHickey.h"
 #include <commands/Shoot.h>
+#include <frc/smartdashboard/SmartDashboard.h>
```
Comment (jasonplatte) [inline src/main/include/RobotContainer.h:23]: You have this in both the header and the source file. Should remove. tisk tisk.

**Review by NWSchallock — DISMISSED** [review]
LGTM

File: src/main/include/RobotContainer.h [hunk 2]
```diff
@@ -86,13 +86,17 @@ class RobotContainer {
...
-  WPI_TalonFX mshooterleft{kShooterLeft};
-  WPI_TalonFX mshooterright{kShooterRight};
-
-  frc::DoubleSolenoid mhood{kHoodSolenoidFoward, kHoodSolenoidReverse};
-  frc::DoubleSolenoid mlatch{kLatchSolenoidFoward, kLatchSolenoidReverse};
+    
+  Shooter::Components mShooterComponents{
+    WPI_TalonFX{1},
+    WPI_TalonFX{2},
+    frc::DoubleSolenoid{2,5},
```
Comment (jasonplatte) [inline src/main/include/RobotContainer.h]: Put in the corresponding constants here instead of the ID. Ex. "kShooterLeft".

**Review by jasonplatte — APPROVED** [review]
LGTM

**Pull request merged**